integration/test/agent_util.go

```diff
@@ -95,6 +95,17 @@ func RunShellScript(path string, args ...string) {
 	}
 }
 
+func RunPowerShellScript(path string, args ...string) {
+	ps, _ := exec.LookPath("powershell.exe")
```

Comment:
I don't think that we should be ignoring an error here.

Reply:
It should be safe to ignore error here since there is another [error gating when execution](https://github.com/aws/amazon-cloudwatch-agent/pull/478/files#diff-914f4c18aa0763094ec05c567b3d2a2481e25cd9e970fe0a3e081a118012de0eR104-R106)